### ISSUE TITLE
Show full output in text editor link is not accessible

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#177841]
Testing #177719

When there's too much output, this link uses `title` to explain what it does:

<img width="76" alt="image" src="https://user-images.githubusercontent.com/2193314/226686833-14725e0c-6620-4ce2-a338-291612e4a3b2.png">

<img width="205" alt="image" src="https://user-images.githubusercontent.com/2193314/226686815-c80a64c3-934f-41a0-bdc1-f5691f6a9314.png">

This isn't accessible, you should set `aria-label` such that it's read by a screen reader.
